pipeline/article_scraper/AmisScraper/pipelines.py

```diff
@@ -21,47 +23,46 @@ def __init__(self):
 
     def process_item(self, item, spider):
         if item['link'] in self.ids_seen:
-            raise DropItem("Duplicate item found: %s" % item)
+            raise DropItem('Duplicate item found: %s' % item)
         else:
             self.ids_seen.add(item['link'])
             return item
 
 
 class SanitizeArticlePipeline(object):
 
-    def __init__(self):
```

Comment:
Are we dealing with stop words in the proper text analysis? there is a lot of garbage we could easily pre-filter...